fraim/reporting/reporting.py

```diff
@@ -13,6 +13,7 @@
 
 from fraim.outputs.sarif import ArtifactContent, PhysicalLocation, Region, Result, SarifReport
 
+# TODO: Relate to the config logger
```

Comment:
maybe implement this as part of the feature since we're in here. You can just pass the logger directly into the `generate_html_report` since it's a class method and Reporting never actually gets initialized.

Reply:
Started on this, but it unraveled. There is no config object in the report generation, and entirely different args.